src/main/java/jep/python/PyBuiltins.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2023 JEP AUTHORS.
+ *
+ * This file is licensed under the the zlib/libpng License.
+ *
+ * This software is provided 'as-is', without any express or implied
+ * warranty. In no event will the authors be held liable for any
+ * damages arising from the use of this software.
+ * 
+ * Permission is granted to anyone to use this software for any
+ * purpose, including commercial applications, and to alter it and
+ * redistribute it freely, subject to the following restrictions:
+ * 
+ *     1. The origin of this software must not be misrepresented; you
+ *     must not claim that you wrote the original software. If you use
+ *     this software in a product, an acknowledgment in the product
+ *     documentation would be appreciated but is not required.
+ * 
+ *     2. Altered source versions must be plainly marked as such, and
+ *     must not be misrepresented as being the original software.
+ * 
+ *     3. This notice may not be removed or altered from any source
+ *     distribution.
+ */
+package jep.python;
+
+import jep.Interpreter;
+
+/*
+ * Interface for accessing some of the functions of the Python builtins module from Java.
+ * More information on specific functions can be found at https://docs.python.org/3.5/library/functions.html
```

Comment:
Why not link to the current documentation https://docs.python.org/3/library/functions.html?  Also will this generate javadoc with an html link in it, or do you need an `<a href="http://...">` tag?  See MemoryManager's javadoc.

Reply:
I have been working off of the 3.5 documentation because we don't want to expose any features that are not available in older versions of python that we still support, however given the limited number of functions we support and how annoying it would be to update when we no longer support old versions I have changed it to 3 and made it a proper link.